tfhe/examples/utilities/print_doc_bench_parameters.rs

```diff
@@ -1,59 +1,40 @@
-    PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64,
-    PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64,
-    PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64,
-};
+use tfhe::shortint::parameters::multi_bit::tuniform::p_fail_2_minus_64::ks_pbs_gpu::PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
 
 pub fn main() {
     println!("CPU Integer parameters:\n");
-    println!("{}", PARAM_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64.name());
-    println!("{PARAM_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64:?}");
+    println!("{}", PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64.name());
+    println!("{PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64:?}");
 
     println!("\n\n===========================================================================\n\n");
 
     println!("CUDA GPU Integer parameters:\n");
     println!(
         "{}",
-        PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64.name()
+        PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64.name()
     );
-    println!("{PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64:?}");
+    println!("{PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64:?}");
 
     println!("\n\n===========================================================================\n\n");
 
     println!("CPU PBS parameters:\n");
-    for param in [
-        PARAM_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64,
-        PARAM_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64,
-        PARAM_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64,
-        PARAM_MESSAGE_4_CARRY_4_KS_PBS_GAUSSIAN_2M64,
-    ] {
-        let bits = (param.message_modulus.0 * param.carry_modulus.0).ilog2();
-        println!("Precision {bits} bits");
-        println!("{}", param.name());
-        println!("{param:?}\n");
-    }
+    let param = PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+    let bits = (param.message_modulus.0 * param.carry_modulus.0).ilog2();
+    println!("Precision {bits} bits");
+    println!("{}", param.name());
+    println!("{param:?}\n");
 
     println!("\n===========================================================================\n\n");
 
     println!("CUDA GPU PBS parameters:\n");
-    for param in [
-        PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64,
-        PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64,
-        PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64,
-    ] {
-        let bits = (param.message_modulus.0 * param.carry_modulus.0).ilog2();
-        println!("Precision {bits} bits");
-        println!("{}", param.name());
-        println!("{param:?}\n");
-    }
+    let param = PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+    let bits = (param.message_modulus.0 * param.carry_modulus.0).ilog2();
+    println!("Precision {bits} bits");
+    println!("{}", param.name());
```

Comment:
this may not work if the parameter is not "named", i.e. added to some keycache macro

Reply:
These a=parameters are now defined in the keycache macro.